.gitignore

```diff
@@ -33,6 +33,7 @@ ct-libc/
 /desktop/dist/
 /desktop/main.prod.js
 /desktop/main.prod.js.map
+/desktop/config.json
```

Comment:
add config.json to gitignore

Reply:
Do you mean replace it with just `config.json`?
Global ignore of any config.json isn't dangerous?
The name is quite general, and we are talking about specifically the file that is created within the dev run.